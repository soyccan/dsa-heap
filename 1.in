7 10
add 12 65 2
drop 3
add -18 21 1
add 90 68 5
add 17 47 5
quota 1 2
print 5
add 14 -5 2
quota 12 3
drop 1
