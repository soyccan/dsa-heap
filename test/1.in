100 100
add 23 17 14 
add 85 50 73 
quota 90 1 
add 86 36 68 
quota 31 0 
quota 16 32 
add 30 62 56 
add 93 5 32 
print 70 
quota 26 39 
print 98 
add 98 72 98 
add 25 38 79 
quota 58 36 
add 59 57 60 
print 73 
drop 5 
drop 46 
drop 87 
drop 5 
quota 63 31 
quota 63 82 
quota 61 6 
print 53 
print 66 
add 44 24 68 
add 35 77 52 
print 51 
drop 36 
quota 92 87 
add 22 92 81 
drop 76 
drop 1 
drop 8 
drop 16 
print 67 
print 1 
add 95 26 20 
print 13 
drop 39 
add 1 85 29 
drop 77 
add 58 51 60 
print 89 
add 74 42 50 
drop 56 
drop 69 
add 17 23 76 
add 1 48 51 
drop 98 
drop 7 
print 75 
add 37 4 4 
print 56 
add 39 97 57 
quota 92 7 
print 48 
drop 68 
add 95 43 52 
add 44 53 63 
print 51 
drop 66 
print 29 
drop 40 
drop 58 
add 94 26 79 
print 83 
add 35 43 43 
add 21 63 60 
add 7 65 28 
print 18 
add 10 69 98 
add 24 79 82 
quota 65 92 
quota 59 71 
add 3 6 80 
drop 49 
print 74 
drop 90 
print 91 
quota 52 19 
quota 70 63 
print 91 
quota 44 16 
quota 26 33 
drop 71 
drop 70 
drop 78 
quota 0 10 
print 50 
quota 90 24 
print 77 
print 77 
quota 87 78 
print 2 
quota 42 16 
quota 58 9 
add 43 19 39 
quota 41 33 
print 66 
