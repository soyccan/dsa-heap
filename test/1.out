23 added to printer 14 with priority 17
85 added to printer 73 with priority 50
quota 90 set for 1
86 added to printer 68 with priority 36
quota 31 set for 0
quota 16 set for 32
30 added to printer 56 with priority 62
93 added to printer 32 with priority 5
no jobs in 70
quota 26 set for 39
no jobs in 98
98 added to printer 98 with priority 72
25 added to printer 79 with priority 38
quota 58 set for 36
59 added to printer 60 with priority 57
85 printed on printer 73
no jobs in 5
no jobs in 46
no jobs in 87
no jobs in 5
quota 63 set for 31
quota 63 set for 82
quota 61 set for 6
no jobs in 53
no jobs in 66
44 added to printer 68 with priority 24
35 added to printer 52 with priority 77
no jobs in 51
no jobs in 36
quota 92 set for 87
22 added to printer 81 with priority 92
no jobs in 76
no jobs in 1
no jobs in 8
no jobs in 16
no jobs in 67
no jobs in 1
95 added to printer 20 with priority 26
no jobs in 13
no jobs in 39
1 added to printer 29 with priority 85
no jobs in 77
58 added to printer 60 with priority 51
no jobs in 89
74 added to printer 50 with priority 42
30 dropped on printer 56
no jobs in 69
17 added to printer 76 with priority 23
1 added to printer 51 with priority 48
98 dropped on printer 98
no jobs in 7
no jobs in 75
37 added to printer 4 with priority 4
no jobs in 56
39 added to printer 57 with priority 97
quota 92 set for 7
no jobs in 48
44 dropped on printer 68
95 added to printer 52 with priority 43
44 added to printer 63 with priority 53
1 printed on printer 51
no jobs in 66
1 printed on printer 29
no jobs in 40
no jobs in 58
94 added to printer 79 with priority 26
no jobs in 83
35 added to printer 43 with priority 43
21 added to printer 60 with priority 63
7 added to printer 28 with priority 65
no jobs in 18
10 added to printer 98 with priority 69
24 added to printer 82 with priority 79
quota 65 set for 92
quota 59 set for 71
3 added to printer 80 with priority 6
no jobs in 49
no jobs in 74
no jobs in 90
no jobs in 91
quota 52 set for 19
quota 70 set for 63
no jobs in 91
quota 44 set for 16
quota 26 set for 33
no jobs in 71
no jobs in 70
no jobs in 78
quota 0 set for 10
74 printed on printer 50
quota 90 set for 24
no jobs in 77
no jobs in 77
quota 87 set for 78
no jobs in 2
quota 42 set for 16
quota 58 set for 9
43 added to printer 39 with priority 19
quota 41 set for 33
no jobs in 66
